src/Store.js

```diff
@@ -219,26 +223,37 @@ class Store {
     amount = amount || this.options.maxHistory
     fetchEntryTimeout = fetchEntryTimeout || this.options.fetchEntryTimeout
 
-    const localHeads = await this._cache.get(this.localHeadsPath) || []
-    const remoteHeads = await this._cache.get(this.remoteHeadsPath) || []
-    const heads = localHeads.concat(remoteHeads)
+    try {
```

Comment:
Do we still need the try/catch block?

Reply:
We do, but it does catch any other errors that might happen besides a ReadError, and we can upgrade the `console.warn` to a `console.error` and still not throw the whole process

Reply:
What issue is happening without this? Same failure in the persistency tests?

Reply:
"close while loading" and replication status tests fail since Store.close is called before Store.load